.github/scripts/tkn_check_parse.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# removes `actions's` default -e flag as it breaks the logic
+set +e
+
+setup() {
+    curl -LO "https://github.com/tektoncd/cli/releases/download/v${TKN_CLI_VERSION}/tektoncd-cli-${TKN_CLI_VERSION}_Linux-64bit.deb" &>/dev/null
+    sudo dpkg -i ./tektoncd-cli-${TKN_CLI_VERSION}_Linux-64bit.deb >/dev/null
+}
+
+is_yaml() {
+    FILE=`basename $1`
+    DIRNAME=`dirname $1`
+    [ ${DIRNAME##*/} == "workflows" ] && return 1
+    [ "${FILE#*.}" == "yaml" ] && return 0 || return 1
```

Comment:
Same as above, I don't think this is needed, because the changeset will only include files ending with `.yaml`. Also, aren't the returns redundant? Wouldn't `[ "${FILE#*.}" == "yaml" ]` do effectively the same thing?

Reply:
The check has been removed, so resolving this thread.